### PR TITLE
review: Cachebust

### PR DESCRIPTION
**Pull request by TobiTenno**
### Warframe Hub Pull Request
---
**Summary (short):** Bust caches using query strings

---
**Description:** Using gulp cachebust, provide unique hashes for files so that if one changes, and thus the hash changes, it will appear as a different url for the browser.

We're trying StackExchange's method detailed here: https://meta.stackexchange.com/questions/112182/how-does-se-determine-the-css-and-js-version-parameter

---
**Fixes issue (include link):** None, just an ongoing caching struggle.

---
**Mockups, screenshots, evidence:** Deployment

---

(Check one)
- [x] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

**Comment by TobiTenno** [comment]
ladies and gentle-tenno, this is why you should just dev in an ide that has linting built in, cause i'm just causing excess commits at this point XD

**Comment by TobiTenno** [comment]
@EricSihaoLin whenever you have time

File: views/partials/dependencies.hbs [hunk 1]
```diff
@@ -1,87 +1,88 @@
...
+      theme.attr('href', "/css/bootstrap-default.min.css?v={{lookup sums 'css/bootstrap-default.min.css'}}");
+      themecustom.attr('href', "/css/main.eidolon.css?v={{lookup sums 'css/main.eidolon.css'}}");
+      theme.toggleClass('eidolon');
+      break;
+    default:
+      theme.attr('href', "/css/bootstrap-default.min.css?v={{lookup sums 'css/bootstrap-default.min.css'}}");
+      themecustom.attr('href', "/css/main.css?v={{lookup sums 'css/main.css'}}");
+      break;
+  }
+  const sums = JSON.parse("{{json sums}}".replace(/&quot;/ig, '"'));
```
Comment (TobiTenno) [inline views/partials/dependencies.hbs:73]: this makes the sums object available for themes

**Review by EricSihaoLin — APPROVED** [review]

**Pull request merged**